common/sgx/verifier.c

```diff
@@ -69,7 +69,7 @@ static void _free_claim(oe_claim_t* claim)
     oe_free(claim->value);
 }
 
-oe_result_t sgx_attestation_plugin_free_claims_list(
+oe_result_t oe_sgx_free_claims_list(
```

Comment:
I'm not 100% sure about the naming convention, but my assumption is that "oe_sgx_..." is typically used for OE SDK core libraries for SGX. Here the function seems to be attestation plugin specific. Isn't the old name better?